include/infinispan/hotrod/Query.h

```diff
@@ -21,14 +21,36 @@ namespace client {
 // jpqlstring field has been renamed to querystring
 // in the .proto file.
 // This class mantains backward compatibility
+/**
+ * This class represent a query request.
+ */
 class QueryRequest : public QueryRequestProtobuf
 {
 public:
   inline bool has_jpqlstring() const { return has_querystring(); }
   inline void clear_jpqlstring() { clear_querystring(); }
+  /**
+   * Use querystring() instead
```

Comment:
I know Deprecated is used by Java, but do you want to use something like that for these comments?